web/packages/teleport/src/User/UserContext.tsx

```diff
@@ -0,0 +1,139 @@
+/**
```

Comment:
Why separate this from `storeUser` ?

Reply:
I'd like to move `storeUser` over this user context. `storeUser` extends `Store`, which isn't really an ideal React-like way to manage data. After this PR is merged, I'll combine the two.